docs/ml/upload-model.md

```diff
@@ -59,11 +64,16 @@ Also, if you selected **Public**, it should be publicly visible in the [Viam reg
 {{<imgproc src="/ml/select-existing-model.png" resize="400x" alt="Select from your existing models.">}}
 
 1. Select the model you would like to update from your existing models.
-2. Select **Next steps** to continue.
-3. Add an updated `.tflite` model file.
-4. (Optional) Add an updated `.txt` label file.
-   This should include the label names as you provided them in training, with one name per line.
-5. Click **Upload model**.
+1. Click **Next steps** to continue.
+1. Select the **Model framework** or the type of model.
+   TensorFlow Lite, TensorFlow, PyTorch, and ONNX model frameworks are currently supported.
+   If you choose a TensorFlow Lite or ONNX model framework, you add a `.txt` label file.
+   This should include the label names you provided in training, with one name per line.
+
+   If you choose a TensorFlow model framework, you should upload it in the [SavedModel format](https://github.com/tensorflow/tensorflow/blob/master/tensorflow/python/saved_model/README.md#the-savedmodel-format).
+
+1. Upload the files required for your model framework.
```

Comment:
It looks like these steps got out of order?

Same suggestion as above:
```suggestion
1. Select the **Model framework** or the type of model.
   TensorFlow Lite, TensorFlow, PyTorch, and ONNX model frameworks are currently supported.
1. Upload the files required for your model framework:
   - For a TensorFlow Lite or ONNX model framework,  upload a `.txt` label file containing the label names you provided in training, with one name per line.
   - For a TensorFlow model framework, upload it in the [SavedModel format](https://github.com/tensorflow/tensorflow/blob/master/tensorflow/python/saved_model/README.md#the-savedmodel-format).
```
As above, consider changing this to "upload a model file...." or whatever is correct, if applicable